models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -38,18 +38,12 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
   params$restart <-c() ## This will be filled with some restart coefficient if above ground wood is in the state variables.
 
   #### PEcAn Standard Outputs
-  if ("GWBI" %in% var.names) {
-    forecast[[length(forecast) + 1]] <- udunits2::ud.convert(mean(ens$GWBI),  "kg/m^2/s", "Mg/ha/yr")
-    names(forecast[[length(forecast)]]) <- c("GWBI")
-  }
-  
```

Comment:
Unclear why GWBI was removed. This quantity is needed to assimilate tree ring data

Reply:
added GWBI back in

Reply:
added GWBI back in